admin/orders.php

```diff
@@ -908,11 +940,16 @@ function couponpopupWindow(url) {
         <div class="row noprint"><?php echo zen_draw_separator('pixel_trans.gif', '1', '5'); ?></div>
         <div class="row noprint">
           <div class="formArea">
-              <?php echo zen_draw_form('statusUpdate', FILENAME_ORDERS, zen_get_all_get_params(array('action')) . 'action=update_order', 'post', 'class="form-horizontal"', true); ?>
+              <?php echo zen_draw_form('statusUpdate', FILENAME_ORDERS, zen_get_all_get_params(array('action','language')) . 'action=update_order&language=' . $order->info['language_code'], 'post', 'class="form-horizontal"', true); ?>
             <div class="form-group">
                 <?php echo zen_draw_label(TABLE_HEADING_COMMENTS, 'comments', 'class="col-sm-3 control-label"'); ?>
               <div class="col-sm-9">
-                  <?php echo zen_draw_textarea_field('comments', 'soft', '60', '5', '', 'id="comments" class="form-control"'); ?>
+                  <?php echo zen_draw_textarea_field('comments', 'soft', '60', '5', '', 'id="comments" class="form-control"');
+                  // remind admin user of the order/customer language in case of writing a comment.
```

Comment:
I haven't explored this, and I'm a bit reluctant to encourage rich-text in these fields, but if you add the `.editorHook` class to this textarea and have CKEditor enabled, it's possible we could also link it to the right language so that it offers spellcheck and UI in the corresponding language expected for the field (this already works for product descriptions).

Reply:
I'm not getting CKeditor loading on product descriptions...

Reply:
> I'm not getting CKeditor loading on product descriptions...

Requirement: choose the CKEditor from the Editor choice before entering the product-edit
<img width="1039" alt="Screen Shot 2020-05-03 at 8 35 58 PM" src="https://user-images.githubusercontent.com/404472/80929908-c42e0480-8d7d-11ea-99a2-0cad2ba5f413.png">

Reply:
I had done that, it didn't display the editor. I left the computer on sleep.
This morning, woke it up, the timeout showed/was logged out.  I logged in, was redirected to order details page. editor was showing.
Disabled the admin switch, editor off.
Enabled the admin switch, editor still off.
Logged off and on, editor still off.

Reply:
as far I can see, this is missing from the head, to allow the editor
```
      <?php
      if ($action === 'edit' && $editor_handler !== '') {
          include ($editor_handler);
      }
      ?>
```

Reply:
Oh, right. Yes, that would need to be added to the Order page for the editor to be usable there.

Reply:
This is a nice addition.

Reply:
Done. Only spellcheck options for textareas to consider now...best as another PR.